examples/dynamic-data.js

```diff
@@ -4,14 +4,14 @@ import {MultiPoint, Point} from '../src/ol/geom.js';
 import TileLayer from '../src/ol/layer/Tile.js';
 import OSM from '../src/ol/source/OSM.js';
 import {Circle as CircleStyle, Fill, Stroke, Style} from '../src/ol/style.js';
+import {getVectorContext} from '../src/ol/render';
```

Comment:
We should be consistant and always have `.js` in our import.
Please cherry-pick:  https://github.com/openlayers/openlayers/commit/0eaccbb0eb97fe59258377d74b4f29155913e6ff

Reply:
Done in 2eede7f46591937d4bdcec16b54cbd23da15283e.